source/partials/_subscribe_to_blog.html.erb

```diff
@@ -1,13 +1,12 @@
 <div class='subscribe-to-blog'>
-   <div class='subscribe-title'>
-     <p>Subscribe to the GoCD blog</p>
-   </div>
-   <div>
-     Subscribe and stay connected with the GoCD Blog. We send a monthly update of what's new in the world of continuous delivery, devops and software development practices.
-   </div>
-   <div>
+<div class="row">
+  <div class="col-md-4">
+   <h3 class='subscribe-title'> Subscribe to the GoCD blog</h3> 
```

Comment:
Can this say - "Subscribe to our newsletter, The Pipeline, for DevOps and Continuous Delivery news in your inbox". OR (if there is less space, can it say "Subscribe to our newsletter - The Pipeline."